dashboard/routings/measurements.py

```diff
@@ -108,21 +164,31 @@ def get_boxplot_per_endpoint():
     endpoints = [str(e.endpoint) for e in get_endpoints()]
 
     data = []
-    for e in endpoints:
-        values = [c.execution_time for c in get_data_per_endpoint(e)]
-        if len(e) > 16:
-            e = '...' + e[-14:]
-        data.append(go.Box(x=values, name=e))
+    for endpoint in endpoints:
+        values = [c.execution_time for c in get_data_per_endpoint(endpoint)]
+        if len(values) == 0:
+            continue
+        data.append(go.Box(
+            x=values,
+            name=endpoint,
+            marker=dict(
+                color=get_color(endpoint)
+            )
+        ))
+
+    if len(data) == 0:
+        return None
 
     layout = go.Layout(
-        autosize=False,
-        width=900,
+        autosize=True,
         height=350 + 40 * len(endpoints),
         plot_bgcolor='rgba(249,249,249,1)',
```

Comment:
shouldn't this rgba(249,249,...) be declared a constant, I see it repeated quite often...